docs/Pitfalls.asciidoc

```diff
@@ -36,3 +36,20 @@ user can share the environment without problems.
 This approach will lead to a problem when the openqa package is updated, since the
 directory permissions will be changed again, nothing a `chmod -R g+rwx /var/lib/openqa/`
 and `chgrp -R openqa /var/lib/openqa` can not fix.
+
+== Performance impact
+
+openQA workers can cause high I/O load, especially when creating VM snapshots.
+The impact therefore gets more severe when +MAKETESTSNAPSHOTS+ is enabled.
```

Comment:
As I'm looking at the rendered version:

+ is not changing the rendering at all - asciidoc quick reference has also no mentioning of this. I guess we want to use `` as monospace for these variables? (unrelated to this PR actually, we markup openqa variables inconsistently all over)

Reply:
It works for me when I transform the .asciidoc into .html with `asciidoc`. Also http://www.methods.co.nz/asciidoc/userguide.html#_text_formatting mentions it for monospace so … wut?

Reply:
http://open.qa/docs/#_networking_in_openqa is the only place where I see monospace - and it's using backticks. And for reference I checked http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#formatted-text

Reply:
https://github.com/powerman/asciidoc-cheatsheet/blob/master/README.adoc also talks about +mono+ and works on github while our ++s don't. very strange

Reply:
Found it! http://asciidoctor.org/news/2014/08/12/asciidoctor-1-5-0-released/#means-passthrough - your document was updated last 2013 - 2014 changed the meaning of + to not affect rendering

Reply:
Well, I have a change prepared replacing all '+' with '`' now but I think the use of `+` is safer so I like to keep that.

Reply:
well, you might want + *and ` meanwhile.